Sources/SwiftDocCUtilities/Action/Actions/PreviewAction.swift

```diff
@@ -84,6 +86,7 @@ public final class PreviewAction: Action, RecreatingContext {
     ///     is performed.
     /// - Throws: If an error is encountered while initializing the documentation context.
     public init(
+        host: String,
```

Comment:
1. duplicate the original init and add host parameter.
2. mark the original init as deprecate.

Reply:
Like I commented [here](https://github.com/apple/swift-docc/pull/713#discussion_r1336099818) I think it's sufficient to provide a default value here to remain _source_ compatible. It won't be _ABI_ compatible but we don't require that.

Reply:
Yeah. Ethan's PR #254 is removing parameters. So he deprecated the old and add a new one.
For this PR, this is just an addition. If we only cares about source compatible, it is indeed enough to provide a default value.